Fix codestyle issue and changelog

bitwise-operators opened the pull request:
style: move private method to correct position
docs: moved new feature to 1.4.0 header in changelog

With this, it should be safe to publish and release a new '1.4.0' tag to trigger Packagist.

Thanks!

Jeroen-G commented:
Could you help out and in [ci.yml](https://github.com/Jeroen-G/autowire/blob/main/.github/workflows/ci.yml) change 

```
on: 
    push:
```

to 

```
on: 
    push:
    pull_request:
```

so codestyle checks are run in pull requests again. 🙏

bitwise-operators commented:
Done. 

Fixing that also revealed that there was insufficient test coverage on the constructor of Electrician, so I added two extra tests for that, and changed it over to a custom Exception instead of Webmozart\Assert.

Jeroen-G commented:
Sweet!

Jeroen-G commented:
Tagged 1.4.0
